### PR TITLE
DOCS: First pass at synching up backup/restore content across both guides

### PR DESCRIPTION
**Pull request by VMWareJennyBeth**
Staging here: https://jenny-dp.sc2-04-pcf1-apps.oc.vmware.com/7-0/admin_guide/managing/backup-main.html

### CODE REVIEW

**Review by lenapivotal — APPROVED** [review]
Looks good to me. 
One thing I'm not sure about. When we generate the OSS guide from the GPDB docs, all the Tanzu Greenplum branding will remain, as it's not conditionalized...  Should it be?

**Comment by VMWareJennyBeth** [comment]
> Looks good to me.
> One thing I'm not sure about. When we generate the OSS guide from the GPDB docs, all the Tanzu Greenplum branding will remain, as it's not conditionalized... Should it be?

You mean conditionalized using otherprops?

**Comment by VMWareJennyBeth** [comment]
> Looks good to me.
> One thing I'm not sure about. When we generate the OSS guide from the GPDB docs, all the Tanzu Greenplum branding will remain, as it's not conditionalized... Should it be?

Wait, I think it IS conditionalized. I'm seeing "otherprops" wherever I see "Tanzu Greenplum" .

File: gpdb-doc/dita/admin_guide/managing/backup-s3-plugin.xml [hunk 1]
```diff
@@ -5,36 +5,95 @@
...
-        href="#topic_ur2_fsn_ndb/s3-plugin-config" format="dita"/>.</p>
-    <p>If you perform a backup operation with the <codeph>gpbackup</codeph> option
-        <codeph>--plugin-config</codeph>, you must also specify the <codeph>--plugin-config</codeph>
-      option when you restore the backup with <codeph>gprestore</codeph>. </p>
+    <section>
+      <title>Prerequisites</title>
+      <p>Using Amazon S3 to back up and restore data requires an Amazon AWS account with access to
+        the Amazon S3 bucket. These are the Amazon S3 bucket permissions required for backing up and
+        restoring data:<ul id="ul_hh5_3mn_h4b">
+          <li><systemoutput>Upload/Delete</systemoutput> for the S3 user ID that uploads the
```
Comment (dyozie) [inline gpdb-doc/dita/admin_guide/managing/backup-s3-plugin.xml:21]: change systemoutput to codeph.  I don't know if systemoutput will render differently with our tools

File: gpdb-doc/dita/admin_guide/managing/backup-s3-plugin.xml [hunk 2]
```diff
@@ -5,36 +5,95 @@
...
+            files</li>
+          <li><systemoutput>Open/Download</systemoutput> and <systemoutput>View</systemoutput> for
+            the S3 user ID that accesses the files</li>
+        </ul></p>
+      <p>For information about Amazon S3, see <xref href="https://aws.amazon.com/s3/" format="html"
```
Comment (dyozie) [inline gpdb-doc/dita/admin_guide/managing/backup-s3-plugin.xml:26]: Better to refer them to "the Amazon S3 documentation" or something similar in the hyperlink.

File: gpdb-doc/dita/admin_guide/managing/plugin_api/restore_data_subset.xml [hunk 3]
```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE topic
+  PUBLIC "-//OASIS//DTD DITA Composite//EN" "ditabase.dtd">
+<topic id="topic1">
+  <title id="pa137116">restore_data_subset</title>
+  <body>
+    <p>Plugin command to stream a filtered dataset from the remote storage
+      system to <codeph>stdout</codeph> .</p>
```
Comment (dyozie) [inline gpdb-doc/dita/admin_guide/managing/plugin_api/restore_data_subset.xml:8]: Remove extra space before period

File: gpdb-doc/dita/admin_guide/managing/plugin_api/restore_data_subset.xml [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+            <codeph>--exclude-table</codeph>, <codeph>--include-table-file</codeph>,
+            or <codeph>&#x2011;&#x2011;exclude-table-file</codeph>) are specified on the 
+            <codeph>gprestore</codeph> command line.</li>
+          <li>The <varname>plugin_config_file</varname> specifies the
+            <codeph>restore_subset: "on"</codeph> property setting.</li>
+        </ul>
+      <p><codeph>gprestore</codeph> invokes the <codeph>restore_data_subset</codeph> 
+        plugin command with an <varname>offsets_file</varname> that it automatically
+        generates based on the filters specified. The <codeph>restore_data_subset</codeph>
+        implementation should extract the start and end byte offsets for each relation
```
Comment (dyozie) [inline gpdb-doc/dita/admin_guide/managing/plugin_api/restore_data_subset.xml:35]: This would be better broken into a bullet list.

**Review by dyozie — APPROVED** [review]

**Pull request merged**